src/Globe/Globe.js

```diff
@@ -187,9 +186,11 @@ Globe.prototype.setLayerVisibility = function(id, visible) {
     this.layersConfiguration.setLayerVisibility(id, visible);
 
     var cO = function(object) {
```

Comment:
You add a root object in `createFeatureLayer` 
```js
Globe.prototype.createFeatureLayer = function(name){
 var featureLayer = new Layer();
 featureLayer.setName(name);
 var dataLayer = new THREE.Object3D();
 featureLayer.add(dataLayer);// <---- Here your add a root object
 this.add(featureLayer);
 return featureLayer;
};
```
and Here you add feature to root node
```js
Globe.prototype.addFeature = function(options){
    var layerId = options.layerId;
    var layer = this.getFeatureLayerByName(layerId);
    if((layerId != undefined) && (options.geometry != undefined) && (layer != undefined)){
           var object  = tool.processingGeoJSON(options.geometry);
           layer.children[0].add(object); // <---- you add feature to root node
    }
};
```
So you can use `featureLayer.children[0].visible` to switch the visibility layer
```js
var featureLayer = map.getFeatureLayerByName(layer);
featureLayer.children[0].visible = visible;
```